From: David Sterba <dsterba@suse.cz>
Date: Mon, 18 Nov 2013 14:24:20 +0100
Patch-mainline: 3.14
Git-commit: 43d87fa23154d135a2a1006bc6656ae73ae84190
Subject: [PATCH] btrfs: reserve no transaction units in
 btrfs_feature_attr_store

Added in patch "btrfs: add ability to change features via sysfs",
modifications to superblock don't need to reserve metadata blocks when
starting a transaction.

Signed-off-by: David Sterba <dsterba@suse.cz>
Signed-off-by: Josef Bacik <jbacik@fb.com>
Signed-off-by: Chris Mason <clm@fb.com>
---
 fs/btrfs/sysfs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/btrfs/sysfs.c
+++ b/fs/btrfs/sysfs.c
@@ -150,7 +150,7 @@ static ssize_t btrfs_feature_attr_store(
 	btrfs_info(fs_info, "%s %s feature flag",
 		   val ? "Setting" : "Clearing", fa->kobj_attr.attr.name);
 
-	trans = btrfs_start_transaction(fs_info->fs_root, 1);
+	trans = btrfs_start_transaction(fs_info->fs_root, 0);
 	if (IS_ERR(trans))
 		return PTR_ERR(trans);
 
